ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -27,8 +28,14 @@ export interface ListingCardContentProps {
   tableHeader?: ListingCardHeader
   tableSubheader?: ListingCardHeader
 }
+
+export interface CardTag extends ImageTag {
+  shadeType?: AppearanceShadeType
+}
+
 export interface ListingCardProps {
-  cardTags?: ImageTag[]
+  cardTags?: CardTag[]
+  homeType: HomeTypeEnum
```

Comment:
Changes look great thank you! ✨ I think with this new set of changes we can remove the `homeType` prop here?